modules/accredited_representative_portal/app/services/power_of_attorney_request_service/create.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module PowerOfAttorneyRequestService
```

Comment:
Whoops. Need one more nesting under `AccreditedRepresentativePortal`. File will have to migrate too.